README.md

```diff
@@ -1,18 +1,32 @@
-# cli
+# Welcome to urfave/cli
 
 [![Run Tests](https://github.com/urfave/cli/actions/workflows/cli.yml/badge.svg)](https://github.com/urfave/cli/actions/workflows/cli.yml)
 [![Go Reference](https://pkg.go.dev/badge/github.com/urfave/cli/v3.svg)](https://pkg.go.dev/github.com/urfave/cli/v3)
 [![Go Report Card](https://goreportcard.com/badge/github.com/urfave/cli/v3)](https://goreportcard.com/report/github.com/urfave/cli/v3)
 [![codecov](https://codecov.io/gh/urfave/cli/branch/main/graph/badge.svg?token=t9YGWLh05g)](https://codecov.io/gh/urfave/cli)
 
-cli is a simple, fast, and fun package for building command line apps in Go. The
-goal is to enable developers to write fast and distributable command line
-applications in an expressive way.
+urfave/cli is a **declarative**, simple, fast, and fun package for building command line tools in Go featuring:
+
+- nestable commands with alias and prefix match support
+- flexible and permissive help system
+- dynamic shell completion for `bash`, `zsh`, `fish`, and `powershell`
+- `man` and markdown format documentation generation
+- input flag types for primitives, slices of primitives, time, duration, and others
```

Comment:
The terminology "primitives" and "slices of primitives" seems unique here. Simple types is better. Why would it be a feature?

Reply:
I like "simple types" :+1: How would you like to see it worded?

Reply:
Depends on what purpose does this feature serve. Why I would need type autoconversion at all? It may bloat the library.

Reply:
I don't follow 😅 Maybe it would help for me to define the audience more clearly (?) I'm attempting to speak to a software engineer who has worked with Go and knows about the standard library `flag` package.

Reply:
@abitrolly I'm in favor of replacing "primitive" with "simple" 👍🏼 I wasn't following what you meant by:

> Depends on what purpose does this feature serve. Why I would need type autoconversion at all? It may bloat the library.

Reply:
@meatballhat my first language is Python, so the comment was that the library exposes too much types. Uint, Float64, which are too specific for a command line API. And if I understand it correctly, if these types are not used, they will still take space in importing app.